docs/src/usage/authentication.md

```diff
@@ -0,0 +1,86 @@
+# Authentication
+
+```
+rover-auth 0.0.0
+🔐  Manage authentication
+
+USAGE:
+    rover auth <SUBCOMMAND>
```

Comment:
I would love to see a discussion about the name for people to give feedback on it! I don't know / think this PR is the right place to do that given its size though

Reply:
should i make an issue or a decision record (or both)?

Reply:
(making an internal decision record)